### DISCUSSION TITLE
RowSpan Table @itwin/itwinui-react

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/itwinjs-core#5519] [Q&A]
Hello good morning, I should create a table within the viewer that groups rows together, there is the possibility of using the rowspan with the <Table> control

I am attaching an example in xls that I should emulate with the <Table> control

Thanks for the precious help
<img width="364" alt="RowSpan" src="https://github.com/iTwin/itwinjs-core/assets/57446652/5f952545-c7ec-409b-a88e-6e3b4b1e0dd8">

**Comment by pratikshan85** [comment]
@Paolo-lab answer to your question was already answered to the below discussion

https://github.com/iTwin/iTwinUI/discussions/1280
